docs/visual-basic/language-reference/data-types/byte-data-type.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Byte Data Type (Visual Basic)"
-ms.date: 04/20/2017
+ms.date: 01/31/2018
```

Comment:
you're already in the future @rpetrusha? 😄

Reply:
We must look forward, not backward, @mairaw!